layers/+lang/haskell/funcs.el

```diff
@@ -22,7 +22,59 @@
     (haskell-navigate-imports)
     (haskell-mode-format-imports)))
 
-;; Dante Functions
+
+;; Completion setup functions
+
+(defun spacemacs-haskell//setup-backend ()
+  "Conditionally setup haskell backend."
+  (message "Setup haskell backend to %s" haskell-completion-backend)
+  (pcase haskell-completion-backend
+    (`ghci (spacemacs-haskell//setup-ghci))
+    (`intero (spacemacs-haskell//setup-intero))
+    (`dante (spacemacs-haskell//setup-dante))
+    (`ghc-mod (spacemacs-haskell//setup-ghc-mod))))
+
+(defun spacemacs-haskell//setup-company ()
+  "Conditionally setup haskell backend."
+  (pcase haskell-completion-backend
+    (`ghci (spacemacs-haskell//setup-ghci-company))
+    (`intero (spacemacs-haskell//setup-intero-company))
+    (`dante (spacemacs-haskell//setup-dante-company))
+    (`ghc-mod (spacemacs-haskell//setup-ghc-mod-company))))
+
```

Comment:
This is not an empty line, it's `^L`.

Reply:
:see_no_evil:

Reply:
😸I was reading my PR and saw these double-empty lines. Was like - what the line?!